include/Note.h

```diff
@@ -107,6 +107,16 @@ class LMMS_EXPORT Note : public SerializingObject
 	Note( const Note & note );
 	~Note() override;
 
+	// Note types
+	enum class Type
+	{
+		Regular = 0,
+		Step
+	};
```

Comment:
```suggestion
	// DO NOT modify existing values - would break project file compatibility
	enum class Type
	{
		Regular = 0,
		Step    = 1
	};
```
You could add a warning to future devs, just in case

Reply:
I don't think this needs a special comment - the same goes for any enum whose numerical value is saved to the project file, of which there are quite a few. (I'd like to switch to string values for this reason, but that's a discussion for another time.)